tests/main/views/test_users.py

```diff
@@ -2132,26 +2161,58 @@ def setup(self):
         super(TestAdminEmailCheck, self).setup()
         self.app.config['DM_ALLOWED_ADMIN_DOMAINS'] = ['bananas.org']
 
-    def test_email_address_is_required(self):
+    # deprecated
+    def test_email_address_is_required_get(self):
         response = self.client.get('/users/valid-admin-email')
         assert response.status_code == 400
 
-    def test_invalid_email_is_not_ok(self):
+    def test_email_address_is_required_post(self):
+        response = self.client.post('/users/valid-admin-email')
+        assert response.status_code == 400
```

Comment:
This test will also need a content type.

Reply:
@katstevens good spot. This is fixed in https://github.com/alphagov/digitalmarketplace-api/pull/1037/commits/6456d091ca3b62f4e53b6133ff0ee61c16066c3b.